lib/ui/pages/coursetable/course_table_page.dart

```diff
@@ -695,6 +700,13 @@ class _CourseTablePageState extends State<CourseTablePage> {
     RouteUtils.launchRollCallDashBoardPageAfterLogin();
   }
 
+  Future<void> _showCameraPage(CourseInfoJson courseInfo) async {
+    CourseMainJson course = courseInfo.main.course;
+    String courseId = course.id;
+    Get.back();
+    RouteUtils.launchCameraPage(courseId);
```

Comment:
This can be simplified.

```suggestion
    Get.back();
    RouteUtils.launchCameraPage(course.id);
```